vite/src/lib/phosphor-icons.tsx

```
@@ -0,0 +1,33 @@
+import { ComponentProps } from "react";
+
+export const PhosphorIcons = icons as unknown as IconEntry[];
+
+type IconId = keyof typeof IconComponents;
+
+export function PhosphorIcon({
+  id,
+  fallbackId,
+  ...props
+  // eslint-disable-next-line @typescript-eslint/no-wrapper-object-types
+}: { id: IconId | String; fallbackId?: string } & IconProps) {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)